components/textbook-beta/SummerSchoolCoursePages.vue

```diff
@@ -0,0 +1,340 @@
+<template>
+  <section class="summer-school-course-pages-section">
+    <div>
+      <h2>
+        Course pages
+      </h2>
+      <nav class="summer-school-course-pages-section__nav">
+        <ul class="courses">
+          <li
+            v-for="{ label, segment, url } in courses"
+            :key="label"
+            class="course"
+          >
+            <BasicLink
+              class="course__entry"
+              :class="{ active: label === activeCourseLabel }"
+              :segment="segment"
+              :url="url"
+              @mouseenter="selectCourse(label)"
```

Comment:
@korgan00, could you please elaborate a little bit on this?

Reply:
if you use the index in the `selectCourse`, something like `selectCourse(courseIndex)`.
Then `activeCourse` that currently is...
```
  get activeCourse () {
    const activeCourse = this.courses.find(
      ({ label }) => label === this.activeCourseLabel
    )
    return activeCourse || null
  }
```

... could be something like...

```
  get activeCourse () {
    return this.courses[this. activeCourseIndex] || null
  }
```

Also, it could prevent some future errors if two lectures have the same label.

Reply:
@korgan00 - I didn't ignore this suggestion, I agree actually. We just merged to unblock future work that depends on this PR... 

Just wanted to comment to acknowledge your suggestion 👍🏽